include/envoy/network/address.h

```diff
@@ -17,6 +16,22 @@ namespace Envoy {
 namespace Network {
 namespace Address {
 
+/**
+ * Result holds the rc and errno values resulting from a system call.
+ */
+struct Result {
+
+  /**
+   * The return code from the system call.
+   */
+  int rc;
```

Comment:
Struct/Class member variables have a _ postfix (e.g., int foo_;).
https://github.com/envoyproxy/envoy/blob/master/STYLE.md

Reply:
Addressed in bb2841e.